src/Nordea/Resources/Icons.elm

```diff
@@ -378,173 +378,495 @@ pdf attrs =
-            , Svg.path [ d "M307.45 125.781C307.45 125.781 299.45 157.211 285.45 178.181", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
-            , Svg.path [ d "M308.08 126.561C308.08 126.561 301.83 168.501 303.9 172.561", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
-            , Svg.path [ d "M308.55 127.16C308.55 127.16 306.55 160.01 311.9 162.57", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
-            , Svg.path [ d "M268.21 161.851C268.21 161.851 275.69 161.101 274.87 168.851C274.87 168.851 285.87 164.691 289.37 171.661C289.37 171.661 296.75 162.921 303.56 165.601C303.56 165.601 305.61 158.791 309.94 159.451", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
-            , Svg.path [ d "M280.6 155.191C280.6 155.191 284.95 154.821 284.77 159.191C284.77 159.191 292.54 155.691 294.77 161.241C294.77 161.241 299.01 155.701 304.17 157.451C304.17 157.451 306.05 153.631 309.01 155.451", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
-            , Svg.path [ d "M290.22 146.53C290.22 146.53 293.1 145.48 293.22 149.03C293.22 149.03 297.38 147.53 299.38 150.41C299.38 150.41 301.98 148.09 305.03 149.41C305.478 149.013 306.034 148.758 306.627 148.68C307.221 148.602 307.824 148.703 308.36 148.97", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
-            , Svg.path [ d "M297.53 137.85C297.973 137.848 298.402 138.004 298.741 138.29C299.08 138.576 299.306 138.973 299.38 139.41C299.38 139.41 301.95 138.51 302.97 140.41C302.97 140.41 304.76 139.21 306.08 140.23C306.366 139.93 306.747 139.737 307.158 139.684C307.57 139.632 307.987 139.722 308.34 139.94", SvgAttrs.stroke "#111111", SvgAttrs.strokeWidth "0.25", SvgAttrs.strokeMiterlimit "10" ] []
+            [ width "32", height "32", viewBox "0 0 32 32", fill "none" ]
+            [ Svg.path
+                [ d "M10 29C10.2431 28.9996 10.4777 28.9107 10.66 28.75L24.37 16.75C24.5853 16.5601 24.7086 16.287 24.7086 16C24.7086 15.713 24.5853 15.4398 24.37 15.25L10.66 3.24997C10.2431 2.91417 9.63568 2.96626 9.28202 3.36814C8.92837 3.77002 8.95393 4.37913 9.34001 4.74997L22.2 16L9.34001 27.25C8.92631 27.6147 8.88604 28.2456 9.25001 28.66C9.43958 28.8758 9.71279 28.9996 10 29Z"
+                , fill (Colors.toString Colors.deepBlue)
+                ]
+                []
             ]
         ]
 
 
-iconContainer : List (Attribute msg) -> List (Html msg) -> Html msg
-iconContainer =
-    styled div
-        [ display inlineFlex
-        , alignItems center
-        , children [ everything [ flex (int 1) ] ]
-        , flexShrink zero
+chevronLeft : List (Attribute msg) -> Html msg
+chevronLeft attrs =
+    iconContainer attrs
+        [ Svg.svg
+            [ width "32", height "32", viewBox "0 0 32 32", fill "none" ]
```

Comment:
I think explicit view and height should be removed so that the size can be controlled by css in each use case. (I see there are a few other icons here as well that have it set)
```suggestion
            [ viewBox "0 0 32 32", fill "none" ]
```